oozie-to-airflow/mappers/subworkflow_mapper.py

```diff
@@ -106,11 +108,10 @@ def _parse_config(self):
     def convert_to_text(self):
         return render_template(template_name=self.template, **self.__dict__)
 
-    @staticmethod
-    def required_imports() -> Set[str]:
+    def required_imports(self) -> Set[str]:
         return {
             "from airflow.utils import dates",
             "from airflow.contrib.operators import dataproc_operator",
             "from airflow.operators.subdag_operator import SubDagOperator",
-            f"from subdag_test import sub_dag",
+            f"from subdag_{self.app_name} import sub_dag",
```

Comment:
I'm afraid of what will happen when there are two Workflow nodes.

Reply:
Proposal: I think the simplest solution will be to import the whole module and in the DAG used <module>.sub_dag ? That would simplify it as we would not have to generate different function names.

Reply:
Done